src/rpc/server.cpp

```diff
@@ -471,6 +487,69 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
     return out;
 }
 
+/**
+ * Process positional arguments into an Object of named arguments, based on the
+ * passed-in specification for the RPC call's arguments.
+ */
+static inline JSONRPCRequest transformPositionalArguments(const JSONRPCRequest& in, const std::vector<std::string>& argNames)
+{
+    JSONRPCRequest out = in;
+    if (in.params.size() > argNames.size()) {
+        // Too many params, so just trigger help
+        out.fHelp = true;
+        return out;
+    }
+    out.params = UniValue(UniValue::VOBJ);
+    out.params.pushKV(ARGS_WERE_POSITIONAL, true);
```

Comment:
It's intended for #11413

Reply:
How? Should be added there?

Reply:
It should probably fail if a named-args caller specifies `conf_target` with the explicit fee mode, or if `feerate` is specified with the automatic fee modes, so it would need to check if the duplication is merely because of positional arguments being used or not.